sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,278 @@
+__author__ = 'TDWilkinson'
+
+"""
+AIA response functions by integrating ChiantiPy
+
+    - analyze AIA instrument response that tells efficiency of channel using instrument properties
+    - then output a wavelength response
+    - use spectral model (synthetic spectra) - to get ion emissivities as a function of temp/density
+    - obtain temperature response based on chianti spectral contribution functions (emissivity)
+
+
+other important variables:
+effective area
+ion emissivity
+
+electron density
+temperature array
+
+# utilize sunpy/instr/tests/test_aia.py
+"""
+
+import numpy as np
+import pandas as pd
+from scipy.io import readsav
+
+import sunpy
+import sunpy.data.test as test
+
+
+def get_function(ion, emissivity, temperature, density, optional = None):
+    """
+    General format of functions where this is the statement of usefulness of function. If these were fits images, it would search the header for relevant information.
+
+    Parameters
+    ----------
+    variable1 : what module of import is used.
+        explanation
+    optional: (optional) string
+        A string specifying optional variable
+        e.g. strings
+
+    Returns
+    -------
+    output: tuple
+    """
+    pass
+# think: how to test
+
+def get_data_from_aia_inst_file(path_to_file):
+    """
+    This definition reads the instrument file aia_V6_all_fullinst file which was obtained from ssw_aia_response_data inside
+    ssw_aia_response_genx.tar.gz (an output file of SolarSoft Ware (SSW)). It will extract the instrument data and save
+    it a dataframe file for easier access.
+
+     Parameters
+    ----------
+    :param path_to_file: string, the path location that leads to ssw_aia_response_data/aia_V6_all_fullinst.
+
+    Returns
+    -------
+    output: channel_properties.csv, a comma separated dataframe file with arrays for each property per channel
+
+    """
+
+    # access outer dictionary from .genx file
+    ssw_dict = readsav(path_to_file)
+
+    # access inner dictionary from .genx file
+    data = ssw_dict['data']
+
+    # define wanted properties
+    indexes = ['wave', 'effarea', 'units', 'geoarea', 'scale', 'platescale', 'numfilters',  'wavemin', 'wavestep', 'wavenumsteps', 'wavelog', 'filtersincludemesh', 'meshtrans', 'usecontam', 'contamthick',  'usephottoelec','elecperev','usephottodn', 'elecperdn', 'useerror', 'fp_filter', 'ent_filter', 'primary', 'secondary', 'ccd', 'contam', 'cross_area']
+    channels = ['A94', 'A131', 'A171', 'A193', 'A211', 'A304', 'A335']
+
```

Comment:
You can actually pull all the channels from the .genx file itself so no need to hardcode them. Something like,
```Python
>>> import scipy.io
>>> ssw_genx = scipy.io.readsav('/path/to/ssw_file.genx')
>>> ssw_genx['channels'][0]
```